### ISSUE TITLE
ECDSA/ECC certificate support

### ISSUE THREAD

**Issue by adisheshsm** [pingidentity/ldapsdk#113]
Hi, 
does ldapsdk support ECDSA/ECC client certificates during startTLS?

thanks and regards,
Adishesh

**Comment by dirmgr** [comment]
Yes, as long as the underlying JVM supports using those certificates. I just tested it with Java 7, which is the oldest Java version that the LDAP SDK supports, and it worked just fine.
